test/spec/spectest.cpp

```diff
@@ -586,9 +602,9 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
         return;
       }
       case CommandID::Action: {
-        const auto &Action = Cmd["action"s];
-        const auto &Expected = Cmd["expected"s];
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
+        const simdjson::dom::object Action = Cmd["action"].get_object();
```

Comment:
Use reference instead of copying a new data structure.

Reply:
Same as above